modules/text_server_adv/text_server_adv.cpp

```diff
@@ -3806,6 +3812,10 @@ void TextServerAdvanced::_font_draw_glyph(const RID &p_font_rid, const RID &p_ca
 				if (ffsd->textures[fgl.texture_idx].dirty) {
 					ShelfPackTexture &tex = ffsd->textures.write[fgl.texture_idx];
 					Ref<Image> img = tex.image;
+					if (fgl.from_svg) {
+						// same as the "fix alpha border" process option when importing SVGs
```

Comment:
```suggestion
						// Same as the "fix alpha border" process option when importing SVGs.
```

Dirty textures can be updated in multiple functions, so `fix_alpha_edges` should be applied there as well.
```
_font_get_glyph_texture_rid
_font_get_glyph_texture_size
_font_draw_glyph
_font_draw_glyph_outline
```